src/main/resources/application.properties

```diff
@@ -8,4 +8,13 @@ spring.datasource.driver-clas-name=org.postgresql.Driver
 #spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 
 spring.datasource.testWhileIdle=true
-spring.datasource.validationQuery=SELECT 1
\ No newline at end of file
+spring.datasource.validationQuery=SELECT 1
+
+#Security
+#Google login
+spring.security.oauth2.client.registration.google.client-id=520695556063-hi6gtn4715qn7ob251045a328cnlmbaj.apps.googleusercontent.com
+spring.security.oauth2.client.registration.google.client-secret=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
+
+#Facebook login
+spring.security.oauth2.client.registration.facebook.client-id=917487869325725
+spring.security.oauth2.client.registration.facebook.client-secret=156a7024a0c18d0afcfd14ef318f623e
```

Comment:
Secret keys are sensitive data and pushing them to github is forbidden. Use .env file to store secrets and then add this file to .gitignore.